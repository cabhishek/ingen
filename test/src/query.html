<!--
{
  "layout": "default",
  "title": "Iterators Test",
  "query": {
    "type": "article",
    "tag": "foo"
  }
}
-->

<h2>Query Test</h2>
<ol>
  {{#query}}
    <li>
      <a href="{{ permalink }}">{{title}}</a><br>
      on: {{date}}
    </li>
  {{/query}}
</ol>
