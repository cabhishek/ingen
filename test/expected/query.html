<!DOCTYPE html>
<html>
  <head>
    <title>InGen</title>
  </head>
  <body>
    <header>Iterators Test</header>

    <main>
<h2>Query Test</h2>
<ol>

    <li>
      <a href="/articles/article-three/">Article Three</a><br>
      on: 2013-10-16
    </li>

    <li>
      <a href="/articles/article-one/">Article One</a><br>
      on: 2013-10-03
    </li>

</ol>
</main>
    <footer>&copy; 2013</footer>

  </body>
</html>
